README.md

```diff
@@ -3,16 +3,22 @@
 This repo holds information about subnet chains that exist on the Avalanche
 network.
 
+## To add your subnet to this list, please make a pull request that adds the following: 
+
 Each chain in the `chains` folder may contain the following files:
-- `chain-information.json`: General information about the chain such as the
-  name, vm configuration, and token configuration.
+- `genesis.json`: To get started, use the
+  [Avalanche CLI](https://github.com/ava-labs/avalanche-cli) to enter your desired configuration parameters and copy over the resulting file using the `export` command.
+- `chain-information-rpc.json`: Minimum information about the chain for a node to be able to connect to your network.
+
+
+### Additional Explorer Information
+- `chain-information.json`: Complete information about a network to connect to the [Avalanche Explorer](https://subnets.avax.network/).
 - `token-list.json`: A list of token contracts that exist on the chain with
   information about each. These tokens have been verified using the contract
   verifier at https://subnets.avax.network/tools/verify-contract.
-- `gensesis.json`: For chains configured using
-  [Subnet EVM](https://github.com/ava-labs/subnet-evm), this contains the
-  genesis configuration parameters.
-- `upgrade.json`: Contains network upgrades for the chain. For more
-  information, see 
+
+
+### Optional Information
+- `upgrade.json`: Contains network upgrades for the chain, *required* for all nodes if implemented. For more information, see 
   https://docs.avax.network/subnets/subnet-upgrade#network-upgrades.
-- `airdrop.json`: Contains a list of addresses that were airdropped tokens in genesis. Note: this file must be referenced in the chain config and the hash must be specified in genesis.
+- `airdrop.json`: Contains a list of addresses that were airdropped tokens in genesis. Note: this file must be referenced in the chain config and the hash must be specified in genesis. This can also be included in the `genesis.json` file directly, see [Avalanche CLI](https://github.com/ava-labs/avalanche-cli).
```

Comment:
Is this the same as the way that the airdrop feature works in avalanche-cli? There are genesis balances and also the airdrop JSON for larger airdrops that may not fit as easily into the genesis file. I believe the avalanche-cli uses the former.